.docker/Dockerfile.php

```
@@ -9,7 +9,7 @@
     && docker-php-ext-install gmp \
     && docker-php-ext-configure gmp
 
-RUN apk add --no-cache --update clamav clamav-libunrar \
+RUN apk add --no-cache --update clamav clamav-libunrar --repository http://dl-3.alpinelinux.org/alpine/edge/main \
```

Use CDN link instead of direct
```suggestion
RUN apk add --no-cache --update clamav clamav-libunrar --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```